xlators/nfs/server/src/nlm4.c

```diff
@@ -1424,8 +1424,10 @@ nlm4svc_lock_cbk(call_frame_t *frame, void *cookie, xlator_t *this,
         ncf = nlm4_notify_init(cs);
         if (ncf) {
             ncf->frame = copy_frame(frame);
-            ncf->frame->local = ncf;
-            nlm4svc_send_granted(ncf);
+            if(ncf->frame){
```

Comment:
I'd argue that if copy_frame() failed, something is really really bad - out of memory for example, and you need to bail out.
This does fix the coverity issue perhaps, but not the real issue.

Reply:
While fixing this issue I've looked how this kind of situation is handled throughout the code.
In most cases code the code is in the form of:
var = copy_frame(frame);
    if (var == NULL) {
        goto out;
}

Moreover, the method calling this one is ignoring the return value anyway (this probably needs to be fixed, but not in the scope of the Coverity fix as this fix will involve functional change)